src/query/api/v1/handler/prometheus/native/common.go

```diff
@@ -55,6 +56,17 @@ const (
 	formatErrStr = "error parsing param: %s, error: %v"
 )
 
+func shortDur(d time.Duration) string {
+	s := d.String()
+	if strings.HasSuffix(s, "m0s") {
+		s = s[:len(s)-2]
+	}
+	if strings.HasSuffix(s, "h0m") {
+		s = s[:len(s)-2]
+	}
+	return s
+}
+
```

Comment:
This should be checked whether it's still needed at all and maybe dropped (but in later PR)